examples/tools/scm/git/capture_scm/git_capture_scm.rst

```diff
@@ -0,0 +1,146 @@
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ on GitHub:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples2.git
+    $ cd examples2/examples/tools/scm/git/capture_scm
+
+
+There we will find a small "hello" project, containing this ``conanfile.py``:
+
+.. code-block:: python
+
+    from conan import ConanFile
+    from conan.tools.cmake import CMake, cmake_layout
+    from conan.tools.scm import Git
+    from conan.tools.files import load, update_conandata
+
+
+    class helloRecipe(ConanFile):
+        name = "hello"
+        version = "0.1"
+
+        # Binary configuration
+        settings = "os", "compiler", "build_type", "arch"
+        options = {"shared": [True, False], "fPIC": [True, False]}
+        default_options = {"shared": False, "fPIC": True}
+        generators = "CMakeDeps", "CMakeToolchain"
+
+        def export(self):
+            git = Git(self, self.recipe_folder)
+            scm_url, scm_commit = git.get_url_and_commit()
+            self.output.info(f"Obtained URL: {scm_url} and {scm_commit}")
+            # we store the current url and commit in conandata.yml
+            update_conandata(self, {"sources": {"commit": scm_commit, "url": scm_url}})
+
+        def source(self):
+            # we recover the saved url and commit from conandata.yml and use them to get sources
+            git = Git(self)
+            sources = self.conan_data["sources"]
+            self.output.info(f"Cloning sources from: {sources}")
+            git.clone(url=sources["url"], target=".")
+            git.checkout(commit=sources["commit"])
+
+        ...
+
+
+We need this code to be in its own Git repository, to see how it works in the real case, so
+please create a folder outside of the ``examples2`` repository, and copy there the contents of the current folder, then:
+
+.. code-block:: text
+
+    $ mkdir /home/myuser/myfolder # or equivalent in other OS
+    $ cp -R . /home/myuser/myfolder # or equivalent in other OS
+    $ cd /home/myuser/myfolder # or equivalent in other OS
+
+    # Initialize the git repo
+    $ git init .
+    $ git add .
+    $ git commit . -m wip
+    # Finally create the package
+    $ conan create .
+    ...
+    hello/0.1: WARN: Current commit 8e8764c40bebabbe3ec57f9a0816a2c8e691f559 doesn't exist in remote origin
+    This revision will not be buildable in other computer
+    hello/0.1: Obtained URL: <local-path>/capture_scm and 8e8764c40bebabbe3ec57f9a0816a2c8e691f559
+    hello/0.1: Copied 1 '.yml' file: conandata.yml
+    hello/0.1: Copied 1 '.py' file: conanfile.py
+    ...
+    hello/0.1: Cloning sources from: {'commit': '8e8764c40bebabbe3ec57f9a0816a2c8e691f559', 'url': '<local-path>/capture_scm'}
+
+Let's explain step by step what is happening:
+
+- When the recipe is exported to the Conan cache, the ``export()`` method executes, running ``scm_url, scm_commit = git.get_url_and_commit()``.
```

Comment:
I think we should add links to the reference for all the Git methods used, like `get_url_and_commit()` I would add one here and maybe a link in the end of the tutorial as a "see also" or "read more" section